letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -1124,12 +1127,14 @@ else
   # package). Phase 2 checks the version of the locally installed certbot.
 
   if [ ! -f "$VENV_BIN/letsencrypt" ]; then
-    if [ "$HELP" = 1 ]; then
-      echo "$USAGE"
-      exit 0
+    if [ -z "$OLD_VENV_PATH" -o ! -f "$OLD_VENV_PATH/bin/letsencrypt" ]; then
```

Comment:
If the user sets `VENV_PATH`, `OLD_VENV_PATH` isn't defined. See [this](https://github.com/certbot/certbot/pull/4970/files#diff-a4881cc6ba1c9dcaeb3962d88a0c369dR26).